### PR TITLE
Put href attribs on project links so hand cursor is displayed.

### PR DESCRIPTION
**Pull request by manwithtwowatches**

### CODE REVIEW

File: public/templates/timelines/toolbar.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="tl-toolbar">
   <div class="tl-toolbar-container">
-    <a ng-if='timeline.USE_MODALS' ng-click="timeline.addPlanningElement()" title="{{I18n.t('js.timelines.new_work_package')}}" class="icon icon-add">
+    <a href="javascript://" ng-if='timeline.USE_MODALS' ng-click="timeline.addPlanningElement()" title="{{I18n.t('js.timelines.new_work_package')}}" class="icon icon-add">
```
Comment (myabc) [inline public/templates/timelines/toolbar.html:3]: @manwithtwowatches why is this `javascript://` here, but an empty attribute in the table template?

Comment (manwithtwowatches) [inline public/templates/timelines/toolbar.html:3]: not sure. i think it might be a left over copypaste job from before. i was thinking about removing all of them but then got lazy. i'll clean them up in the morning.

Comment (myabc) [inline public/templates/timelines/toolbar.html:3]: @manwithtwowatches tbh I don't know which one is better/more correct.

Comment (manwithtwowatches) [inline public/templates/timelines/toolbar.html:3]: i'd probably go for empty attribute. less code:/

Comment (myabc) [inline public/templates/timelines/toolbar.html:3]: @manwithtwowatches it looks like the rest of the toolbar uses `javascript://`. IMO making these consistent is out of scope for this PR.
